### PR TITLE
Update pool decorator, and multiprocessing

### PR DESCRIPTION
**Pull request by Yaoyx**
#535 
#536

### CODE REVIEW

File: cooltools/api/expected.py [hunk 1]
```diff
@@ -1035,6 +1035,10 @@ def per_region_smooth_cvd(
     )
 
     return cvd
+
+def _transformer(p, weight1, weight2):
```
Comment (gfudenberg) [inline cooltools/api/expected.py:1039]: I wonder if this might be better named `_balance_transform`

File: cooltools/api/expected.py [hunk 2]
```diff
@@ -1035,6 +1035,10 @@ def per_region_smooth_cvd(
...
+    return p["count"] * p[weight1] * p[weight2]
+
 # user-friendly wrapper for diagsum_symm and diagsum_pairwise - part of new "public" API
 @pool_decorator
 def expected_cis(
```
Comment (gfudenberg) [inline cooltools/api/expected.py:1044]: would be useful to expose transforms in expected_cis? cc @sergpolly @nvictus

File: tests/data/CN.mm9.10000kb.test_sampled.cool
Comment (gfudenberg) [inline tests/data/CN.mm9.10000kb.test_sampled.cool:1]: seems that a few deletable test files got added to the PR @Yaoyx

**Comment by gfudenberg** [comment]
@dmitrymyl I just invited you as a collaborator for the repo-- let us know if this resolves the issues you encountered!

**Comment by dmitrymyl** [comment]
I checked this version against my profiling notebook -- now obs/exp pups are calculated fast and one can also supply custom map functors (as intended).

Timings:

| Pileup kind | nproc | cooltools (default) | cooltools x multiprocess | cooltools x multiprocessing | cooltools x concurrent.futures | coolpuppy (default) |
|------------:|------:|:-------------------:|:------------------------:|:---------------------------:|:------------------------------:|:-------------------:|
|         OBS |     1 |        9.8 s        |          10.4 s          |            10.4 s           |             10.1 s             |        8.93 s       |
|         OBS |     8 |        2.71 s       |          2.74 s          |            2.74 s           |             2.69 s             |        2.45 s       |
|      OBSEXP |     1 |        11.7 s       |          13.8 s          |            12.6 s           |             14.6 s             |        14.6 s       |
|      OBSEXP |     8 |        5.37 s       |          19.7 s          |            4.48 s           |             5.06 s             |        9.4 s        |

File: cooltools/api/expected.py (hunk 1 above)
Comment (Yaoyx) [reply cooltools/api/expected.py]: Yea, that might be more informative. Updated

File: tests/data/CN.mm9.10000kb.test_sampled.cool
Comment (Yaoyx) [reply tests/data/CN.mm9.10000kb.test_sampled.cool:1]: Removed

**Review by golobor — COMMENTED** [review]
looks good to me!

**Pull request merged**